### PR TITLE
Features menu categories/review box

### PR DESCRIPTION
**Pull request by CamiloGilFranco**
## Summary

- The search panel by category is added to the restaurant menu,

- Fixed the path that is redirected to when clicking on the cards in the main carousel

- Form with validations is created to add comments to restaurants

- Functionality is given to the new comment form to render the new comments

- Syntax errors were fixed in some checkout view components

- Function is added to restaurantCardComponent to solve scroll top bug

### CODE REVIEW

File: src/components/ReviewsComponent/ReviewsComponent.jsx [hunk 1]
```diff
@@ -1,7 +1,58 @@
...
+
+    if (!newReview.comments || newReview.title.length > 255) {
+      setCommentsError(true);
+      isOk = false;
+    } else {
+      setCommentsError(false);
+    }
+
+    if (isOk) {
+      console.log("comentario enviado");
```
Comment (ht1204) [inline src/components/ReviewsComponent/ReviewsComponent.jsx]: is this console.log necessary?

**Review by ht1204 — COMMENTED** [review]
Please, address the comment that I wrote to approve or else if that must be remove, thx.

**Comment by CamiloGilFranco** [comment]
Removed unnecessary console.log

P.S. sorry for the delay

**Review by ht1204 — APPROVED** [review]

File: src/components/ReviewsComponent/ReviewsComponent.jsx (hunk 1 above)
Comment (ht1204) [reply src/components/ReviewsComponent/ReviewsComponent.jsx]: Resolved

**Pull request merged**